projects/cds/src/merchandising/adapters/strategy/cds-merchandising-strategy.adapter.ts

```diff
@@ -3,8 +3,8 @@ import { Injectable } from '@angular/core';
 import { ConverterService } from '@spartacus/core';
 import { Observable } from 'rxjs';
 import { CdsEndpointsService } from '../../../services/cds-endpoints.service';
+import { StrategyAdapter } from '../../connectors/strategy/cds-strategy.adapter';
 import { MERCHANDISING_PRODUCTS_NORMALIZER } from '../../connectors/strategy/converters';
-import { StrategyAdapter } from '../../connectors/strategy/strategy.adapter';
 import { MerchandisingProducts } from '../../model/merchandising.products.model';
 
 const STRATEGY_PRODUCTS_ENDPOINT_KEY = 'strategyProducts';
```

Comment:
change the class name as well to CdsMerchandisingStrategyAdapter please

Reply:
https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5360#pullrequestreview-315591772

from what I said here, 

the connectors/strategy should be named merchandising-strategy.adapter.ts
^ don't forget to change the class name as well please :)

Reply:
My bad, I've renamed the files AND the classes now :-)